From 7ddbab4de18ffa84ccd973131cdb48c962928643 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 14 Aug 2018 16:51:24 +0300
Subject: [PATCH] btl: Add '\n' before printing '1'

It is more distinctive on the console to have the '1' printed
on the new line than to count on some other s/w to print EOL for us.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/btl/uart.s b/board/habanalabs/common/btl/uart.s
index d418ea5..79dc4f9 100644
--- a/board/habanalabs/common/btl/uart.s
+++ b/board/habanalabs/common/btl/uart.s
@@ -24,6 +24,13 @@
 cfg mmPSOC_UART_0_CTRL, 0x3
 
 /* log BTLID\n\r */
+cfg mmPSOC_UART_0_DATA, 0X0A
+nop
+nop
+nop
+nop
+nop
+/* #include "nop.s" */
 cfg mmPSOC_UART_0_DATA, BTLID
 cfg mmPSOC_UART_0_DATA, 0X0A
 cfg mmPSOC_UART_0_DATA, 0X0D
